scripts/update_secrets.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+)
+
+// Check if all the rules that exist in "gitleaks" are included in our list of rules (in secret.go file)
+func main() {
+
+	//Find the latest release of "gitleaks"
+	release, err := fetchLatestRelease()
```

Comment:
You should avoid comments if you can. The code is always changing, but the comments may stay in confuse.
```suggestion
	release, err := fetchGitleaksLatestRelease()
```